<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1282</width>
    <height>703</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="inputLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>输入图像</string>
    </property>
   </widget>
   <widget class="QLabel" name="outputLabel">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>0</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>输出图像</string>
    </property>
   </widget>
   <widget class="QScrollArea" name="inputScrollArea">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>500</width>
      <height>550</height>
     </rect>
    </property>
    <property name="cursor" stdset="0">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustToContents</enum>
    </property>
    <property name="widgetResizable">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>232</y>
       <width>118</width>
       <height>85</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QScrollArea" name="outputScrollArea">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>0</y>
      <width>500</width>
      <height>550</height>
     </rect>
    </property>
    <property name="cursor" stdset="0">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustToContents</enum>
    </property>
    <property name="widgetResizable">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents_2">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>232</y>
       <width>118</width>
       <height>85</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>225</x>
      <y>580</y>
      <width>70</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>原图像</string>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>745</x>
      <y>580</y>
      <width>70</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>处理结果</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QSlider" name="verticalSlider">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>40</y>
      <width>51</width>
      <height>371</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>430</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>缩放系数：</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1282</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>菜单</string>
    </property>
    <addaction name="actionload"/>
    <addaction name="actionexit"/>
    <addaction name="actionWindow"/>
    <addaction name="actionsaveImage"/>
    <addaction name="actionenhance"/>
    <addaction name="actionreverse"/>
    <addaction name="actionturnAround"/>
    <addaction name="actionstepback"/>
    <addaction name="actionrestore"/>
    <addaction name="actiongeoTrans"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionload">
   <property name="text">
    <string>载入图片</string>
   </property>
  </action>
  <action name="actionexit">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="actiontrans">
   <property name="text">
    <string>几何变换</string>
   </property>
  </action>
  <action name="actionWindow">
   <property name="text">
    <string>窗位窗宽(Ctrl+W）</string>
   </property>
  </action>
  <action name="actionsaveImage">
   <property name="text">
    <string>保存图片</string>
   </property>
  </action>
  <action name="actionenhance">
   <property name="text">
    <string>图像增强</string>
   </property>
  </action>
  <action name="actionreverse">
   <property name="text">
    <string>灰度反转</string>
   </property>
  </action>
  <action name="actionturnAround">
   <property name="text">
    <string>左右翻转</string>
   </property>
  </action>
  <action name="actionstepback">
   <property name="text">
    <string>撤销(Ctrl+F)</string>
   </property>
  </action>
  <action name="actionrestore">
   <property name="text">
    <string>恢复图像</string>
   </property>
  </action>
  <action name="actiongeoTrans">
   <property name="text">
    <string>几何变换</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
